### DISCUSSION TITLE
on scanning proceess sometimes Red state turn to Green, and some times not, how can always trun red screen to green by finding familiar camera position?

### DISCUSSION THREAD

**Discussion by damavand1** [introlab/rtabmap#1188] [General]
During room scanning process using the Kinect camera, sometimes the screen turns red.

By repositioning the camera in a previously scanned familiar state, it turns green again and continues its operation. However, sometimes, if the screen turns red and I place the camera in a previously scanned familiar state, it doesn't turn green anymore and remains red permanently. and I force to select "reset odometry" and all my data scanned will be lost

Scanning a room or house at once is not an easy task and requires a high level of precision. If the screen turns red during the scanning process and remains red, everything needs to be restarted from the beginning.

What is the solution?

thanks

**Comment by matlabbe** [comment]
Actually, after hitting "reset odometry", the map scanned so far would "disappear" to show the new mapping session, but the old one is still there and rtabmap still try to find loop closures with it. If a loop closure is found with the previous session, it should reappear. To see an example of that, see the [video](http://www.youtube.com/watch?v=xbKVhZQFKgA) (at 1:30) at the end of this section https://github.com/introlab/rtabmap/wiki/Kinect-mapping#lost-odometry-red-screens
